other/alarms.php

```diff
@@ -192,9 +196,9 @@ function edit_table($row){
     $alarm_id = isset($_GET["alarm_id"]) ? $_GET["alarm_id"] : null;
     $check = isset($_GET["check"]) ? $_GET["check"] : "";
     $no_repeat_interval = isset($_GET["no_repeat_interval"]) ? $_GET["no_repeat_interval"] : 3600;
-    $quiries = isset($_GET["quiries"]) ? $_GET["quiries"] : array_fill(0, NUM_INPUTS, "");
-    $recipients = isset($_GET["recipients"]) ? $_GET["recipients"] : array_fill(0, NUM_INPUTS, "");
-    $parameters = isset($_GET["parameters"]) ? $_GET["parameters"] : array_fill(0, NUM_INPUTS, "");
+    $quiries = isset($_GET["quiries"]) ? $_GET["quiries"] : array_fill(0, NUM_INPUTS + 1, "");
```

Comment:
Can you shed some light on what the "+1" change is about?

Reply:
supposedly an off-by-one error now fixed

Reply:
Exactly. The code had an off-by-error that used to be hidden by PHP in earlier PHP versions.

Reply:
🤦🏻  thanks